### PR TITLE
None option for modal

### PR DESCRIPTION
**Pull request by j-storey5**
Sorry for all the clutter, changes got made when running yarn test and then github wanted to update every file to match the latest test date. All feature changes are in /src/components/Modal/Modal.tsx and .../Modal/Modal.stories.tsx

### CODE REVIEW

File: src/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -39,6 +39,8 @@ const CloseButtonContainer = styled(Button.Container)`
   ${({ closeButtonAttachment }: { closeButtonAttachment: string }) => {
     let distance;
     let position;
+    let display_type = 'flex'; //what is the desired default for the close button's display-type??
```
Comment (aVileBroker) [inline src/components/Modal/Modal.tsx:42]: default distance and position to something here
update display_type to `inline-flex`

change `display_type` to `display`

**Review by aVileBroker — CHANGES_REQUESTED** [review]

File: src/components/Modal/Modal.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ const CloseButtonContainer = styled(Button.Container)`
   ${({ closeButtonAttachment }: { closeButtonAttachment: string }) => {
     let distance;
     let position;
-    let display_type = 'flex'; //what is the desired default for the close button's display-type??
+    let display = 'in-line flex'; //default display type
```
Comment (aVileBroker) [inline src/components/Modal/Modal.tsx:42]: This should be `inline-flex`, which is a display attribute
https://developer.mozilla.org/en-US/docs/Web/CSS/display
```suggestion
    let display = 'inline-flex'; //default display type
```

**Review by aVileBroker — APPROVED** [review]

**Review by aVileBroker — CHANGES_REQUESTED** [review]
accidental approval - still needs inline-flex change

**Review by aVileBroker — APPROVED** [review]

**Comment by j-storey5** [comment]
Merge Success!

**Pull request merged**